ui/packages/consul-ui/app/components/topology-metrics/status/index.hbs

```diff
@@ -1,12 +1,14 @@
-{{#if @noMetricsReason}}
+{{#if (eq @kind 'ingress-gateway')}}
+  <span class="topology-metrics-error">{{t "components.consul.topology-metrics.status.ingress-gateway"}}</span>
+{{else if @noMetricsReason}}
   <span class="topology-metrics-error">
-    Unable to load metrics
+    {{t "components.consul.topology-metrics.status.error"}}
     <span>
       <Tooltip>{{@noMetricsReason}}</Tooltip>
```

Comment:
Could we update this deprecated tooltip component to use the modifier one whilst we in this file? Ideally if you are editing a file and you see these would should be upgrading them as we go (happy for that to be here or in another PR).

Reply:
I tried ```{{tooltip @noMetricsReason}}``` and it did not work. No changes made.